example-app/src/main/java/com/example/dao/TasksDaoImpl.java

```diff
@@ -76,6 +76,19 @@ public Task addTask(@Nonnull final Task task) {
         }
     }
 
+    @Nonnull
+    @Override
+    public Task deleteTask( int id) {
+        sleepMyLittlePrincess();
+        final SQLiteDatabase db = dbHelper.getWritableDatabase();
+        try{
+            db.delete(TaskEntry.TABLE_NAME, TaskEntry._ID + "=?", new String[] {Integer.toString(id)} );
+            return null;
```

Comment:
Probably here you have a warning, because method is marked as @Nonnull and you are returning null.
This method should return void